docs/DatabaseCatalogAPI.yaml

```diff
@@ -324,58 +376,68 @@ components:
         - deprecated
         - inactive
         - development
-      example: inactive
-
-    Dataset:
-      allOf:
-        - $ref: "#/components/schemas/DatasetId"
-        - $ref: "#/components/schemas/DatasetBuilder"
+      example: active
 
-    DatasetBuilder:
+    BasicDataset:
       type: object
       properties:
-        hosted_url:
-          description: The URL of the dataset data as hosted by MobilityData. No authentication required.
-          type: string
-          example: http://www.londontransit.ca/gtfsfeed/google_transit.zip
-        data_type:
-          description: Always `gtfs` for GTFS datasets.
+        id:
+          description: Unique identifier used as a key for the datasets table.
           type: string
-          example: gtfs
+          example: dataset_0
         feed_id:
           description: ID of the feed related to this dataset.
           type: string
           example: feed_0
-        download_date:
-          type: string
-          example: 2020-12-31
-          format: date
-        creation_date:
-          type: string
-          example: 2020-12-31
-          format: date
-        last_update_date:
-          type: string
-          example: 2020-12-31
-          format: date
-        hash:
-          description: A MD5 hash of the dataset.
-          type: string
-          example: a_long_sha1_hash
-        bounding_box:
-          $ref: "#/components/schemas/BoundingBox"
-        features:
-          description: An array of features for this dataset.
-          type: array
-          items:
-            type: string
-            enum:
-              - fares-v2
-              - fares-v1
-              - flex-v1
-              - flex-v2
-              - pathways
-            example: fares-v2
+        data_type:
+          $ref: "#/components/schemas/DataType"
+
+    GtfsDataset:
+      allOf:
+        - $ref: "#/components/schemas/BasicDataset"
+        - type: object
+          properties:
+            legacy_id:
+              description: An optional legacy ID that could be useful for data migration.
+              type: string
+              example: ID_101
+            hosted_url:
+              description: The URL of the dataset data as hosted by MobilityData. No authentication required.
+              type: string
+              example: https://storage.googleapis.com/storage/v1/b/mdb-latest/o/us-maine-casco-bay-lines-gtfs-1.zip?alt=media
+            note:
+              description: A note to clarify complex use cases for consumers.
+              type: string
+            download_date:
+              type: string
+              example: 2020-12-31
+              format: date
+            creation_date:
```

Comment:
Tagging @davidgamez since @jcpitre mentioned you suggested this: 

Is there ever a situation where the `creation_date` for a dataset will be different from our `download date`? If we're not extracting the actual feed and looking at `feed_info.txt`, `calendar.txt` or calendar_dates.txt, I assume this'll always line up with when we compare datasets and download the new one.

Reply:
You are right; as for MVP, we are not parsing any date from the GTFS spec itself. What about if we make it simple for consumers and collapse `download_date`, `creation_date `, and `last_update_date ` on one field?  Keeping only the `date` that MobilityData stores the file.

Reply:
This makes sense to me for datasets! `download_date` is the clearest to me. (`date` might not be specific enough in the future as we add to the MVP.) Any other thoughts on naming?

Reply:
+1 for `download_date`.